src/Interfaces/Options/Shape/IImageShape.ts

```diff
@@ -0,0 +1,12 @@
+export interface IImageShape {
+    src: string;
+    width: number;
+    height: number;
+
+    /**
+     * @deprecated use the new replaceColor instead
+     */
+    replace_color: boolean;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'replace_color' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139404)